README.md

```diff
@@ -146,6 +146,18 @@ Order Multiple Teacher Sets Configuration
 MAXIMUM_COPIES_REQUESTABLE :5  - This is a configuration value in AWS ElasticBeanstalk. In future if anyone want to change the value of maximum teacherset orders, we can update in AWS ElasticBeanstalk Configuration.
 ```
 
+Show Maintenance Banner Configuration
+========================
+```
+SHOW_MAINTENANCE_BANNER: TRUE  - This is a configuration value in AWS ElasticBeanstalk. In future if anyone want to change the value of maintenanct banner, we can update in AWS ElasticBeanstalk Configuration.
```

Comment:
Please change this section to:
```
SHOW_MAINTENANCE_BANNER: TRUE
This parameter can be set in the ElasticBeanstalk AWS console. 
It should be set to the string 'TRUE' to turn on the banner. The banner text is set in the `MAINTENANCE_BANNER_TEXT` parameter, below.

MAINTENANCE_BANNER_TEXT: 'Maintenance banner text' 
This parameter can be set in the ElasticBeanstalk AWS console. 
It should be set to the value which we wish to appear in the banner. It will only appear if the `SHOW_MAINTENANCE_BANNER` parameter above is set to `TRUE`.
```